### ISSUE TITLE
nginx-gen unable to parse nginx.tmpl and not creating /etc/nginx/conf.d/default.conf

### ISSUE THREAD

**Issue by saschwarz** [evertramos/docker-compose-letsencrypt-nginx-proxy-companion#240]
# Error description:
I set `docker context use` to my remote machine and run `./start.sh`.
The docker images download/build and the containers start.
But I was getting errors for all my domains from letsencrypt:

```bash
nginx-letsencrypt    | 2021-01-11 15:14:29,563:INFO:simp_le:1414: Generating new certificate private key
nginx-letsencrypt    | 2021-01-11 15:14:31,503:ERROR:simp_le:1396: CA marked some of the authorizations as invalid, which likely means it could not access http://example.com/.well-known/acme-challenge/X. Did you set correct path in -d example.com:path or --default_root? Are all your domains accessible from the internet? Please check your domains' DNS entries, your host's network/firewall setup and your webserver config. If a domain's DNS entry has both A and AAAA fields set up, some CAs such as Let's Encrypt will perform the challenge validation over IPv6. If your DNS provider does not answer correctly to CAA records request, Let's Encrypt won't issue a certificate for your domain (see https://letsencrypt.org/docs/caa/). Failing authorizations: https://acme-v02.api.letsencrypt.org/acme/authz-v3/9996998067
nginx-letsencrypt    | Challenge validation has failed, see error log.
```

I checked the following

```bash

docker-compose-letsencrypt-nginx-proxy-companion > docker-compose ps
      Name                     Command                 State                              Ports                         
------------------------------------------------------------------------------------------------------------------------
nginx-gen           /usr/local/bin/docker-gen  ...   Restarting                                                         
nginx-letsencrypt   /bin/bash /app/entrypoint. ...   Up                                                                 
nginx-web           /docker-entrypoint.sh ngin ...   Up           x.x.x.x:443->443/tcp, x.x.x.x:80->80/tcp

docker-compose-letsencrypt-nginx-proxy-companion > docker-compose logs nginx-web 
Attaching to nginx-web
nginx-web            | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx-web            | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx-web            | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
nginx-web            | 10-listen-on-ipv6-by-default.sh: error: /etc/nginx/conf.d/default.conf is not a file or does not exist
nginx-web            | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx-web            | /docker-entrypoint.sh: Configuration complete; ready for start up


docker-compose-letsencrypt-nginx-proxy-companion > docker-compose logs nginx-gen
Attaching to nginx-gen
nginx-gen            | 2021/01/11 15:14:22 Unable to parse template: read /etc/docker-gen/templates/nginx.tmpl: is a directory
nginx-gen            | 2021/01/11 15:19:24 Unable to parse template: read /etc/docker-gen/templates/nginx.tmpl: is a directory
nginx-gen            | 2021/01/11 15:19:24 Unable to parse template: read /etc/docker-gen/templates/nginx.tmpl: is a directory
```

TL;DR: It looks like the `nginx.tmpl` isn't being mounted/created correctly(?) when processed by `nginx-gen` so it isn't creating `/etc/nginx/conf.d/default.conf`???

# Steps to reproduce

I set the `IP` in the `.env` file and ran `start.sh`

# Comments

Any suggestions would be greatly appreciated!

**Comment by saschwarz** [comment]
I found if clone the repo on the server and run `start.sh` everything works fine. So you just can't run it via a remote docker.
